### ISSUE TITLE
Discv5 CPU limit

### ISSUE THREAD

**Issue by AgeManning** [sigp/lighthouse#1212]
## Description

Discv5 is currently very permissive with responding to requests and performs multiple query requests in parallel. This draws large CPU resources when connected to a large number of peers. 

Discv5 should be built into its own task as per  #1206 and provide more configuration parameters to allow session limiting and request throttling.

**Comment by AgeManning** [comment]
This is blocked on upgrading the underlying discv5 implementation. Progress is tracked in this PR:
https://github.com/sigp/discv5/pull/14

**Comment by torfbolt** [comment]
Any progress on this, is it still blocked?

**Comment by AgeManning** [comment]
This is underway. I'll make a PR to track this either today or tomorrow.

**Comment by paulhauner** [comment]
#1247 is looking like it solves this. See https://github.com/sigp/lighthouse/pull/1274#issuecomment-645872197

**Comment by AgeManning** [comment]
Resolved in #1247
